content/concepts/payment-system-basics/accounts/multi-signing.md

```diff
@@ -0,0 +1,12 @@
+# Multi-Signing
+
+Multi-signing is one of three ways to authorize transactions for the XRP Ledger, alongside signing with [regular keys and master keys](cryptographic-keys.html). You can configure your address to allow any combination of the three methods to authorize transactions.
+
+Benefits of multi-signing include:
+
+* You can require keys from different devices, so that a malicious actor must compromise multiple machines to send transactions on your behalf.
+* You can share custody of an address between multiple people, each of whom only has one of several keys necessary to send transactions from that address.
+* You can delegate the power to send transactions from your address to a group of people, who can control your address if you are unavailable or unable to sign normally.
+* ... and more.
+
+<!--{# TODO: Expand this and add it to the dactyl-config #}-->
```

Comment:
Should we open a ticket to track this?

Reply:
https://ripplelabs.atlassian.net/browse/DOC-1558